packages/playwright/src/generators/configuration/files/playwright.config.ts.template

```diff
@@ -16,7 +16,7 @@ const baseURL = process.env['BASE_URL'] || '<% if(webServerAddress) {%><%= webSe
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  ...nxE2EPreset(__filename, { testDir: './<>' }),
+  ...nxE2EPreset(__filename, { testDir: './src' }),
```

Comment:
shouldn't this be the directory option passed in?

Reply:
yes it should. changed.